src/sentry/remote_config/docs/protocol.md

```diff
@@ -22,17 +22,68 @@ Retrieve a project's configuration.
 
 **Feature Object**
 
-| Field | Type   | Description                        |
-| ----- | ------ | ---------------------------------- |
-| key   | string | The name used to lookup a feature. |
-| value | any    | A JSON value.                      |
+| Field    | Type           | Description                                                                     |
+| -------- | -------------- | ------------------------------------------------------------------------------- |
+| key      | string         | The name used to lookup a feature.                                              |
+| value    | any            | A JSON value.                                                                   |
+| variants | array[Variant] | Features can have any number of possible variants which alter the return value. |
+
+Features may specify zero or more variants which may alter the evaluation result.
+
+**Variant Object**
+
+| Field   | Type                 | Description                                                                               |
+| ------- | -------------------- | ----------------------------------------------------------------------------------------- |
+| rollout | union[Rollout, null] | An object which contains instructions for rolling out a variant.                          |
+| rules   | array[Rule]          | An array of rules which all must evaluate to true in order to return the variant's value. |
```

Comment:
was gonna suggest `union[array[Rule], null]` but instead we'll just send down an empty array.